src/Silk.Core.Data/MediatR/Guilds/AddGuildRequest.cs

```diff
@@ -26,7 +26,7 @@ public AddGuildHandler(GuildContext db)
 
         public async Task<Guild> Handle(AddGuildRequest request, CancellationToken cancellationToken)
         {
-            var guild = new Guild {Id = request.GuildId, Configuration = new(), Prefix = request.Prefix};
+            var guild = new Guild {Id = request.GuildId, Configuration = new(), ModConfig = new(), Prefix = request.Prefix};
```

Comment:
Should the `Configuration` and `ModConfig` also have their `GuildId` properties set in their object initializers with the `request.GuildId`?

Reply:
I *think* EFCore handles this? It's been like this since forever

Reply:
Ahh ok, cool